docs/dev/android/development-with-mozilla-central.md

```diff
@@ -50,3 +50,65 @@ Make sure to use the correct path pointing to your Glean checkout.
 >
 > See [Replacing the Glean Gradle plugin in mozilla-central](gradle-plugin-in-mc.md).  
 > For replacing `glean_parser` in a local build, see [Substituting `glean_parser`](glean-parser-substitution.md).
+
+## Manual `mavenLocal()`
+
+If the [Substituting projects](#substituting-projects) does not work you can manually switch to using `mavenLocal()`.
+
+In the Glean repository:
+
+1. Bump the version: `bin/prepare-release.sh 65.1.0`
+    * Only increase the minor version to ensure it works transitively
+1. Publish the Gradle project locally: `./gradlew publishToMavenLocal`
+1. Check that the packages are available in `~/.m2/repository`
+
+In `mozilla-central`:
+
+1. In `gradle.properties` remove the line `org.gradle.configuration-cache=true`
```

Comment:
Alternatively, you can skip this if you build Fenix/Focus passing in `--no-build-cache --no-configuration-cache`